File: Android/doraemonkit/src/main/java/com/didichuxing/doraemonkit/kit/network/bean/Response.java
package com.didichuxing.doraemonkit.kit.network.bean;

import java.io.Serializable;

/**
 * 响应bean,不包含内容Body
 */
public class Response  implements Serializable{

    public String url;

    public int status;

    public String headers;

    public String mimeType;

    @Override
    public String toString() {
        return String.format("[%s %d  %s %s]", url, status, headers.toString(), mimeType);
    }
}

File: Android/kotlin/doraemonkit/src/main/java/com/didichuxing/doraemonkit/kit/network/okhttp/bean/Response.kt
package com.didichuxing.doraemonkit.kit.network.okhttp.bean

import java.io.Serializable

/**
 * 响应bean,不包含内容Body
 */
class Response : Serializable {
    var url: String? = null
    var status = 0
    var headers: String? = null
    var mimeType: String? = null
    override fun toString(): String {
        return String.format("[%s %d  %s %s]", url, status, headers.toString(), mimeType)
    }
}